presto-tests/src/main/java/com/facebook/presto/tests/TestedFeature.java

```diff
@@ -21,18 +21,25 @@
 
 public enum TestedFeature
 {
+    DATE_TYPE,
+
     SCAN,
     JOIN,
     ORDER_BY,
-    DELETE,
-
     FILTERED_AGGREGATION,
     AGGREGATION(FILTERED_AGGREGATION),
+    DELETE,
+    INSERT,
 
     CREATE_TABLE,
-
-    METADATA(ADD_COLUMN, DROP_COLUMN, CREATE_TABLE),
-    CONNECTOR(SCAN, METADATA),
+    DROP_TABLE,
```

Comment:
I think it's quite reasonable to assume that any connector supporting CREATE TABLE also supports DROP TABLE. Let's introduce separate constant for DROP_TABLE only when needed.